src/bindings/python/src/openvino/runtime/__init__.py

```diff
@@ -0,0 +1,62 @@
+# Copyright (C) 2021 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+"""openvino module namespace, exposing factory functions for all ops and other classes."""
+# noqa: F401
+
+from pkg_resources import get_distribution, DistributionNotFound
+
+try:
+    __version__ = get_distribution("openvino-core").version
+except DistributionNotFound:
+    __version__ = "0.0.0.dev0"
+
+
+# Openvino pybind bindings and python extended classes
+from openvino.runtime.impl import Dimension
+from openvino.runtime.impl import Function
+from openvino.runtime.impl import Node
+from openvino.runtime.impl import PartialShape
+from openvino.runtime.impl import Layout
+
+from openvino.runtime.ie_api import Core
```

Comment:
@alalek ok, thank you, I'll try it.

```
BTW, Do we have import openvino.Core somewhere?
```
As I know and as I saw in files, no, we don't have `import openvino.Core`

Reply:
@alalek ok, I've tried to add `openvino.Core = Core` inside `openvino/runtime/__init__.py` and also `openvino = sys.modules['openvino']`

But the same problem occurred.
Attach console
![image](https://user-images.githubusercontent.com/20324612/143767167-6e97ef7d-042c-43fa-8955-fa9164363cde.png)

Reply:
Looks like the root `__init__.py` doesn't really import sub-packages during initialization (please add some debug "print" into `runtime/__init__.py` to check that).

---

Can we add dummy subpackages for Core, Tensor, etc?

`openvino/Core/__init__.py`:

```
import openvino

from openvino.runtime.ie_api import Core
openvino.Core = Core
del sys.modules['openvino.Core']
```

Need to check that these cases work with workaround:
- `import openvino.Core`
- `from openvino import Core`

Reply:
We could initialize sub-packages using the `walk_packages` approach:

Root `openvino/__init__.py`:

```.diff
 __path__ = __import__('pkgutil').extend_path(__path__, __name__)

+def _subpackage_import_error(name):
+    print("WARN: Error importing sub-package '%s'" % name)
+    pass
+
+for module_info in __import__('pkgutil').walk_packages(__path__, __name__ + '.', onerror=_subpackage_import_error):
+    print(module_info.name)
+    pass
```

Reply:
@alalek 
I may try but looks like other subpackages like accuracy_checker should also add these lines.
Also one small doubt came from here: https://packaging.python.org/guides/packaging-namespace-packages/#pkgutil-style-namespace-packages
```
The __init__.py file for the namespace package needs to contain only the following:

__path__ = __import__('pkgutil').extend_path(__path__, __name__)
```
As I know, the code after this line is inaccessible. 

BTW, this PR is like quick fix to unblock all, I'll continue to make  `from openvino import Core` work here: https://github.com/openvinotoolkit/openvino/pull/8846

Reply:
CC @slyubimt

Reply:
> the code after this line is inaccessible.

My local `print()` experiment shows that code before and after this line runs properly.
There is no anything special - just regular [Python code](https://github.com/python/cpython/blame/v3.6.0/Lib/pkgutil.py#L500).

However it it critical that all `openvino/__init__.py` files **must have the same content**. Including AC.

---

if we need to initialize "runtime" only (and we perform all necessary initialization in `runtime/__init__.py`), then this would be enough (without `walk_packages`):

```.diff
 __path__ = __import__('pkgutil').extend_path(__path__, __name__)
+__import__(__name__ + '.runtime')  # force initialization
```

---

AC possible workaround for `__init__`:
- use `openvino` -> `openvino_tools` package
- AC's `openvino_tools/__init__.py` - no special requirements except `.extend_path()`
- OV's `openvino/__init__.py`: add second "extend_path" line:
  ```.diff
   __path__ = __import__('pkgutil').extend_path(__path__, __name__)
  +__path__ = __import__('pkgutil').extend_path(__path__, 'openvino_tools')
  +__import__(__name__ + '.runtime')
  ```

Verify (not sure about AC actual name):

- `import openvino.accuracy_checker`
- `from openvino import accuracy_checker`
- `import openvino` without installed `openvino_tools`